packages/footer/components/Newsletter.tsx

```diff
@@ -0,0 +1,239 @@
+import {
+   Box,
+   Button,
+   FormControl,
+   FormErrorMessage,
+   HStack,
+   Input,
+   Stack,
+   Text,
+   Flex,
+   Link,
+} from '@chakra-ui/react';
+import { faCircleCheck, faEye } from '@fortawesome/pro-solid-svg-icons';
+import {
+   Subscription,
+   SubscriptionStatus,
+   useSubscribeToNewsletter,
+} from '../../newsletter-sdk';
+import React from 'react';
+import { FaIcon } from '../../icons';
```

Comment:
Same here... I think we should be using the package name: `@ifixit/icons` instead of a relative path that reaches *outside* the package directory.

Reply:
[32deab5](https://github.com/iFixit/react-commerce/pull/1342/commits/32deab58142a96fc9145cf6932e6a8d825ab2574)